styleguide/derek/_partials/solutions/announcements.ejs

```diff
@@ -0,0 +1,53 @@
+<div class="container-fluid standard-padding bg-light-gray">
+  <div class="container">
```

Comment:
Just to reiterate what we spoke about @WidgetsBurritos , As most of the solutions pattern sections contain a nested <div class="container"> inside of a <div class="container-fluid">,  we should get a group together and come up with a different solution so that we’re all on the same page If we’re going to change the way we layout sections like that. I don’t think it’s a good idea to change that in this PR, it’s starting to get outside the scope of it. It's something that we should address though.